scripts/debian/postinst.in

```diff
@@ -67,6 +67,25 @@ if [ "$1" = "configure" ]; then
             esac
           fi
           clear
+        else
+          case $FALCO_DRIVER_CHOICE in
+              module | kmod )
```

Comment:
I would just enforce proper namings (the same names that we use for falco systemd units for example? Or the ones we use in Falco config)
But i can do that once this is merged, in https://github.com/falcosecurity/falco/pull/2905